.vscode/launch.template.json

```diff
@@ -0,0 +1,59 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+      {
+        "type": "node",
+        "name": "Attach",
+        "port": 9229,
+        "request": "attach",
+        "skipFiles": ["<node_internals>/**"]
+      },
+      {
+        "type": "node",
+        "request": "launch",
+        "name": "Beacon",
+        "skipFiles": ["<node_internals>/**"],
+        "smartStep": true,
+        "program": "${workspaceFolder}/packages/cli/bin/lodestar.js",
+        "args": ["beacon"],
+        "console": "integratedTerminal"
+      },
+      {
+        "type": "node",
+        "request": "launch",
+        "name": "Dev",
+        "skipFiles": ["<node_internals>/**"],
+        "smartStep": true,
+        "program": "${workspaceFolder}/packages/cli/bin/lodestar.js",
+        "args": ["dev"],
+        "console": "integratedTerminal"
+      },
+      {
+        "name": "Test Current File",
+        "type": "node",
+        "request": "launch",
+        "program": "${workspaceFolder}/node_modules/.bin/vitest",
+        "args": ["--run", "${file}", "-t", "${input:testName}"],
+        "cwd": "${workspaceFolder}/${input:packageName}",
+        "console": "integratedTerminal",
+        "skipFiles": ["<node_internals>/**"]
+      }
+    ],
+    "inputs": [
+      {
+        "id": "packageName",
+        "type": "command",
+        "command": "extension.commandvariable.transform",
```

Comment:
it should be possible to make this work without the extension, just using bash

Reply:
Hmm ok. Shall I remove this configuration for now?

Reply:
I am fine with keeping as is, we can update it later